htlcswitch/resolve_empty.go

```diff
@@ -0,0 +1,16 @@
+// +build noresolver
```

Comment:
What is this file for? Building without resolver logic?

Reply:
yes